### DISCUSSION TITLE
How to proxy port to domain subdirectory

### DISCUSSION THREAD

**Discussion by diktus** [NginxProxyManager/nginx-proxy-manager#3143] [General]
Hi,

I have services running on https://domain.com:3670. i want to redirect port 3670 to domain sudirectory like https://domain.com/files/. i have try add custom path /files and forward to HTTPS://IPaddress and port 3670 but this is not working. how to do this correctly for service that require https

Thank You

**Comment by carloda** [comment]
> Hi,
> 
> I have services running on https://domain.com:3670. i want to redirect port 3670 to domain sudirectory like https://domain.com/files/. i have try add custom path /files and forward to [HTTPS://IPaddress](HTTPS://IPaddress) and port 3670 but this is not working. how to do this correctly for service that require https
> 
> Thank You

Did you manage to solve it? I have the same problem.
